featuregate/README.md

```diff
@@ -67,7 +67,10 @@ modeled after the [system used by Kubernetes](https://kubernetes.io/docs/referen
    of the collector.
 
 Features that prove unworkable in the `alpha` stage may be discontinued 
-without proceeding to the `beta` stage.  Features that make it to the `beta` 
-stage will not be dropped and will eventually reach general availability 
-where the `Gate` that allowed them to be disabled during the `beta` stage 
-will be removed.
+without proceeding to the `beta` stage. Instead, they will proceed to the
+`deprecated` stage, which will feature is permanently disabled. A feature gate will
+be removed once it has been `deprecated` for at least 2 releases of the collector.
+
+Features that make it to the `beta` stage will not be dropped and will eventually
```

Comment:
Are we sure that we never revert this? Just curios, what is then the purpose of beta if no roll-back?

Reply:
We can definitely change its meaning, but the current README says this: `Features that prove unworkable in the alpha stage may be discontinued without proceeding to the beta stage.  Features that make it to the beta stage will not be dropped and will eventually reach general availability`.

If we don't want to be so locked in we could remove that wording, but I think the original intent was that if we are going to send it to beta and enable it by default we are going to keep the feature.  We use beta instead of going straight to stable to give users time to adjust to the new feature, not because we think we might get rid of it.

Reply:
I agree that in general sending a feature gate to beta signals an intent to go forward with it, but it's always possible that widespread use while in beta reveals that a feature is untenable or otherwise needs major rework. I think it makes sense to indicate here that beta features are intended to become stable and likely won't be deprecated, but I'm not sure we should lock ourselves into guaranteeing that they become stable.

Reply:
Updated the stance on beta to be less strict and document the rollback procedure.